dart/lib/src/flx_types.dart

```diff
@@ -0,0 +1,152 @@
+import 'dart:typed_data';
+
+enum BitWidth {
+  width8,
+  width16,
+  width32,
+  width64
+}
+
+class BitWidthUtil {
```

Comment:
why wrap these in a class? is that common in Dart?

Reply:
Sadly enum in dart do not support method definitions by default. There is a new feature called extension methods, which allow adding methods to enum definitions, but it was introduced only in Dart 2.7 and after a brief discussion with @dnfield I decided to not use them.